src/main/java/com/epam/ta/reportportal/core/widget/content/loader/MostFailedContentLoader.java

```diff
@@ -111,7 +117,8 @@ private void validateWidgetOptions(Map<String, List<String>> widgetOptions) {
 	 * @return Map of grouped content fields by first part. Expected only one value.
 	 */
 	private void validateContentFields(Map<String, List<String>> contentFields) {
-		BusinessRule.expect(contentFields, notNull()).verify(ErrorType.BAD_REQUEST_ERROR, "Content fields should not be null.");
+		BusinessRule.expect(MapUtils.isNotEmpty(contentFields), equalTo(true))
+				.verify(ErrorType.BAD_REQUEST_ERROR, "Content fields should not be empty");
 		BusinessRule.expect(contentFields.size(), Predicate.isEqual(1))
```

Comment:
as we have change the structure of content fields, we should change this validation with something more suitable

Reply:
and provided more detailed docs on restrictions